main.nf

```diff
@@ -252,21 +252,6 @@ ${summary.collect { k,v -> "            <dt>$k</dt><dd><samp>${v ?: '<span style
 }
 
 
-/*
```

Comment:
Why not keep this and provide an extra parameter that enables users to specify the path to already prefeteched models? This way, we'd have slim containers, flexibility _and_ the possibility to download models if we require them :-)

Reply:
That's weird - do you have the code for that available somewhere? Shouldn't be a big issue to adapt it in a way to keep this out of the container...

Reply:
hmm.. i mean you can try it yourself when you remove the line 'fetch' from the dockerfile and add the process instead the final process 'predict_peptides' which depends on this download will fail.

Reply:
I just added it to the `predict_peptides`step itself to see what happens 👍

Reply:
ok I see - well great that it works.